### DISCUSSION TITLE
How to play chess with the computer on macOS?

### DISCUSSION THREAD

**Discussion by Bohan-Song** [official-stockfish/Stockfish#5218] [Q&A] [answered]
I downloaded the software from the Mac App Store.
I only found the analysis mode. There is no way to play chess with the computer.

**Comment by vondele** [comment, marked as answer]
Stockfish is the chess engine, not the GUI  / App that allows you to play. How to play depends on your GUI see also 

https://github.com/official-stockfish/stockfish/wiki/Download-and-usage#download-a-chess-gui 

for a choice

**Reply by Bohan-Song** [reply]
Oh! Thanks~
